### DISCUSSION TITLE
Copilot doesn't work properly in VSCode

### DISCUSSION THREAD

**Discussion by softfarr** [community/community#18146] [Copilot Conversations]
Hi5 to all. I got email welcoming me to GitHub Copilot Technical Preview. I installed and authorized VSCode extension as shown in the image and I went to test copilot and didn't work. Any suggestion showed. Could anybody help me to fix it? Thanks in advance.

![Screenshot from 2022-06-09 10-05-57](https://user-images.githubusercontent.com/10351296/172881512-dd3ea003-8a63-4979-89a1-efbf04451247.png)
![Screenshot from 2022-06-09 10-05-47](https://user-images.githubusercontent.com/10351296/172881517-571b76e9-dd57-4b63-9da8-ce431494bb05.png)

**Comment by mateonunez** [comment]
Check your JSON settings. You should find this setting (if it is not present, add it):

```json
    "github.copilot.enable": {
        "*": true,
    }
```

Then restart your **Extension Host**

**Reply by softfarr** [reply]
Thanks. It worked!!!
![Screenshot from 2022-06-09 14-00-06](https://user-images.githubusercontent.com/10351296/172924963-84f1b27c-0bdd-4ba7-aeb6-e9ef9a9f0530.png)
![Screenshot from 2022-06-09 14-07-02](https://user-images.githubusercontent.com/10351296/172925055-76f7fd54-6582-4497-b052-0fe8f35d4951.png)

**Comment by ronbentata** [comment]
had the same issue. Reloading the extension once more after installation worked for me. 
now suggestions work great ^_^
